packages/devtools_app/web/flutter_bootstrap.js

```diff
@@ -0,0 +1,46 @@
+{{flutter_js}}
+{{flutter_build_config}}
+
+// Unregister the old custom DevTools service worker (if it exists). It was
+// removed in: https://github.com/flutter/devtools/pull/5331
+function unregisterDevToolsServiceWorker() {
+  if ('serviceWorker' in navigator) {
+    const DEVTOOLS_SW = 'service_worker.js';
+    const FLUTTER_SW = 'flutter_service_worker.js';
+    navigator.serviceWorker.getRegistrations().then(function(registrations) {
+        for (let registration of registrations) {
+            const activeWorker = registration.active;
+            if (activeWorker != null) {
+                const url = activeWorker.scriptURL;
+                if (url.includes(DEVTOOLS_SW) && !url.includes(FLUTTER_SW)) {
+                    registration.unregister();
+                }
+            }
+        }
+    });
+  }
+}
+
+// Bootstrap app for 3P environments:
+function bootstrapAppFor3P() {
+  _flutter.loader.load({
+    serviceWorkerSettings: {
+      serviceWorkerVersion: {{flutter_service_worker_version}},
+    },
+    config: {
+      canvasKitBaseUrl: 'canvaskit/'
+    }
+  });
+}
+
+// Bootstrap app for 1P environments:
+function bootstrapAppFor1P() {
+  _flutter.loader.load({
+    config: {
+      canvasKitBaseUrl: 'canvaskit/'
```

Comment:
Looks like this is a change from the previous implementation, where we did not set the `canvasKitBaseUrl` field for 1P. @elliette do you remember why we needed separate bootstrapping for 1P? And does this look okay to you?

Reply:
Ah, that's just an oversight by me. I can remove that.

Reply:
I forget why the `canvasKitBasUrl` is only required for 3P, but in general the assets are located in slightly different locations for 1P vs 3P which is why there are two different implementations.